llvm/lib/Transforms/Utils/SSAUpdaterBulk.cpp

```diff
@@ -132,8 +125,8 @@ void SSAUpdaterBulk::RewriteAllUses(DominatorTree *DT,
                       << " use(s)\n");
 
     SmallPtrSet<BasicBlock *, 2> DefBlocks;
-    for (auto &Def : R.Defines)
-      DefBlocks.insert(Def.first);
+    for (auto [BB, V] : R.Defines)
```

Comment:
```suggestion
    for (const auto &[BB, V] : R.Defines)
```

Reply:
BB and V are pointers, I don't need a const ref to a pointer

Reply:
Sorry, overlooked, indeed it doesn't matter here.